### PR TITLE
add(fields to pacbio sample)

### PR DESCRIPTION
**Pull request by ChrOertlin**
## Description

closes https://github.com/Clinical-Genomics/add-new-tech/issues/96

### Added

-

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b add-pacbio-orderin -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/models/orders/samples.py [hunk 1]
```diff
@@ -195,9 +195,16 @@ def str_to_int(cls, v: str) -> int | None:
 class PacBioSample(OrderInSample):
     _suitable_project = OrderType.PACBIO_LONG_READ
 
+    buffer: str
     capture_kit: str | None = None
+    container: ContainerEnum | None
```
Comment (islean) [inline cg/models/orders/samples.py:200]:
```suggestion
    container: ContainerEnum
```
I don't think it should be possible to send it as None. One of the values of the enum is "no container"

File: cg/models/orders/samples.py [hunk 2]
```diff
@@ -195,9 +195,16 @@ def str_to_int(cls, v: str) -> int | None:
...
+    container_name: str | None
```
Comment (islean) [inline cg/models/orders/samples.py:201]:
```suggestion
    container_name: str | None = None
```

File: cg/models/orders/samples.py [hunk 3]
```diff
@@ -195,9 +195,16 @@ def str_to_int(cls, v: str) -> int | None:
...
+    name: str
```
Comment (islean) [inline cg/models/orders/samples.py:202]:
```suggestion
```
Already inherited from the OrderInSample

File: cg/models/orders/samples.py [hunk 4]
```diff
@@ -195,9 +195,16 @@ def str_to_int(cls, v: str) -> int | None:
...
     sex: SexEnum = SexEnum.unknown
+    source: str
+    subject_id: str
     tumour: bool
+    volume: str
```
Comment (islean) [inline cg/models/orders/samples.py:207]:
```suggestion
```
Already inherited from the OrderInSample

File: cg/models/orders/samples.py [hunk 5]
```diff
@@ -195,9 +195,16 @@ def str_to_int(cls, v: str) -> int | None:
 class PacBioSample(OrderInSample):
     _suitable_project = OrderType.PACBIO_LONG_READ
 
+    buffer: str
     capture_kit: str | None = None
```
Comment (islean) [inline cg/models/orders/samples.py:199]:
```suggestion
```
I can not see this field in the issue

**Review by islean — APPROVED** [review]
Nice! Also wondering if we need to add some validation but maybe we can make do until the validation project picks this project type up?

**Review by diitaz93 — APPROVED** [review]
👍

**Pull request merged**